test/integration/csv_preview_test.rb

```diff
@@ -236,16 +253,18 @@ class CsvPreviewTest < ActionDispatch::IntegrationTest
     assert page.has_title?("Attachment 2 - GOV.UK")
   end
 
-  def setup_asset_manager(parent_document_url, asset_manager_id, filename = nil, use_special_characters_csv: false, asset_deleted: false, media_code: 200)
+  def setup_asset_manager(parent_document_url, asset_manager_id, filename = nil, malformed: false, nonconvertible: false, asset_deleted: false, media_code: 200)
     asset_manager_response = {
       id: "https://asset-manager.dev.gov.uk/assets/foo",
       parent_document_url:,
       deleted: asset_deleted,
     }
     stub_asset_manager_has_an_asset(asset_manager_id, asset_manager_response, filename)
 
-    csv_file = if use_special_characters_csv
-                 "\xEF\xBB\xBF\"Field 1\",\"Field 2\",\"Field 3 \n(details)\",\"Field 4 (\xC2\xA3m)\"\n\"Value 1\",\"Value 2\",\"Value 3\",\"Value 4 \xC2\xA33.8bn\"\n"
```

Comment:
Although this line looks fancy, the generated error was caused by the first few bytes being outside of the quotation marks. I've simplified the line below.